addons/mail/static/tests/chatter/web/chatter.test.js

```diff
@@ -685,3 +685,21 @@ test("Mentions in composer should still work when using pager", async () => {
     // all records in DB: Mitchell Admin | Hermit | Public user except OdooBot
     await contains(".o-mail-Composer-suggestion", { count: 3 });
 });
+
+test("Picker popover UI size responsiveness", async () => {
+    //Test for small screens
+    patchUiSize({ height: 640, width: 360 });
+    const pyEnv = await startServer();
+    const partnerId = pyEnv["res.partner"].create({});
+    await start();
+    await openFormView("res.partner", partnerId);
+    await click(".btn", { text: "Log note" });
```

Comment:
It is a bad practice to use these bootstrap classes as selectors for your operataions. We prefer `"button", { text: "Log note"}`. You may find other referances in test case files.